### PR TITLE
Issue #7581: Add default config and code examples for EmptyCatchBlock

### PR DESCRIPTION
**Pull request by SGanguly1999**
Closes: #7581

image

`$ cat Hello.java`

```
class Hello {
  public static void main(String args[]) {
    try {

    }
    catch(Exception e) { // violation since catch block is empty

    }
  }
}
```

`$ cat config.xml`

```
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
    <module name="EmptyCatchBlock"/>
  </module>
</module>
```

Output:

```
$ java -jar checkstyle-8.39-all.jar -c config.xml Hello.java
Starting audit...
[ERROR] E:\New folder\Java Dev\src\com\company\Hello.java:10:9: Empty catch block. [EmptyCatchBlock]
Audit done.
Checkstyle ends with 1 errors.

```

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java [hunk 1]
```diff
@@ -53,6 +53,27 @@
  * </li>
  * </ul>
  * <p>
+ *   To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;EmptyCatchBlock&quot;/&gt;
+ * </pre>
+ * <p>
+ *   Example:
```
Comment (strkkk) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java:62]: should be no indentation here

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java [hunk 2]
```diff
@@ -53,6 +53,27 @@
  * </li>
  * </ul>
  * <p>
+ *   To configure the check:
```
Comment (strkkk) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java:56]: same here

File: src/xdocs/config_blocks.xml [hunk 3]
```diff
@@ -447,6 +447,27 @@ public class Test {
...
 
       <subsection name="Examples" id="EmptyCatchBlock_Examples">
+        <p>
+          To configure the check:
+        </p>
+        <source>
+&lt;module name=&quot;EmptyCatchBlock&quot;/&gt;
+        </source>
+        <p>
+        Example:
```
Comment (strkkk) [inline src/xdocs/config_blocks.xml:457]: indentation +2

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java [hunk 4]
```diff
@@ -53,6 +53,27 @@
...
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;EmptyCatchBlock&quot;/&gt;
+ * </pre>
+ * <p>
+ *   Example:
+ * </p>
+ * <pre>
+ * class Hello {
```
Comment (strkkk) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java:65]: please make examples same as ones below to show difference between configs

Also, description says that `By default check allows empty catch block with any comment inside.` please show this

**Review by strkkk — CHANGES_REQUESTED** [review]

**Comment by SGanguly1999** [comment]
@strkkk....I will surely make the changes as soon as possible

**Comment by SGanguly1999** [comment]
@strkkk  there is some mistake in this branch  i am creating a PR attaching this issue on it.

**Pull request closed without merging**